Mathlib/Algebra/Module/Presentation/RestrictScalars.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Algebra.Module.Presentation.DirectSum
+import Mathlib.Algebra.Module.Presentation.Cokernel
+
+/-!
+# Presentation of the restriction of scalars of a module
+
+Given a morphism of rings `A → B` and a `B`-module `M`, we obtain a presentation
+of `M` as a `A`-module from a presentation of `M` as `B`-module,
+a presentation of `B` as a `A`-module (and some additional data).
+
+## TODO
+* deduce that if `B` is a finitely presented as an `A`-module and `M` is
+finitely presented as an `B`-module, then `M` is finitely presented as an `A`-module
+
+-/
+
+universe v u₁ u₂
```

Comment:
Can you use `Type*` instead of naming the universes?

Reply:
Yes.